packages/xfsprogs/0001-mkfs-source-defaults-from-config-file.patch

```diff
@@ -49,7 +49,7 @@ index 3f4455d4..fce7d831 100644
 +	if (stat (DEFAULT_CONFIG_PATH, &defcfg_statbuf) == 0) {
 +		/*
 +		* Set the sb_feat for cli_params as mentioned above. The purpose of
-+		* this is to make sure we inadvertently don't override any default
++		* this is to make sure we don't inadvertently override any default
```

Comment:
This change should be made in the commit that adds the patch, not the commit that updates xfsprogs.